package.json

```diff
@@ -35,6 +35,7 @@
     "react-intl": "^2.1.2",
     "react-router": "^2.4.1",
     "react-treeview": "^0.4.5",
+    "remove-accents": "^0.4.0",
```

Comment:
[Its implementation](https://github.com/tyxla/remove-accents/blob/master/index.js) seems quite trivial and efficient (uses regexes).

The adherence to this library is small since we use it at a 1 place only with a function with a single parameter.